global.json

```diff
@@ -1,5 +1,6 @@
 {
-  "sdk": {
-    "version": "5.0.103"
-  }
-}
\ No newline at end of file
+	"sdk": {
+		"version": "5.0.301",
+		"rollForward": "feature"
```

Comment:
Just for reference, I found [this article by Andrew Lock](https://andrewlock.net/exploring-the-new-rollforward-and-allowprerelease-settings-in-global-json) very helpful in understanding the different `rollForward` policies.

Reply:
and some more reference: [global.json](https://docs.microsoft.com/en-us/dotnet/core/tools/global-json)